content/chainguard/chainguard-images/about/images-categories.md

```diff
@@ -0,0 +1,123 @@
+
+
+## AI Images
+
+Artificial intelligence and machine learning (AI/ML) systems are used in a wide variety of high-stakes applications, including information retrieval, medical research, fraud identification, military operations, autonomous vehicle navigation, and more. If compromised by malicious actors, the consequences could be disastrous and far reaching.
+
+Due to their unique features and uses, these systems often pose a greater risk than traditional software systems. Chainguard offers a suite of CPU and GPU-enabled AI container images which can help to mitigate these risks. Some of these AI images include [NeMo](https://images.chainguard.dev/directory/image/nemo/overview), [PyTorch](https://images.chainguard.dev/directory/image/pytorch/overview), and [TensorFlow](https://images.chainguard.dev/directory/image/tensorflow/overview).
+
+These images are hardened, minimal, and optimized for efficient AI development and deployment. By leveraging Chainguard AI Images, organizations can confidently secure their AI infrastructure, streamline vulnerability management, and maintain high performance with low-to-zero vulnerabilities. Rather than starting with tens, dozens, or hundreds of CVEs in your application or pipeline, you start with a clean slate. 
+
+To learn more about Chainguard's AI images and their uses, we encourage you to check out our course on [Securing the AI/ML Supply Chain](https://courses.chainguard.dev/securing-ai).
+
+## FIPS images
+
+FIPS — or, Federal Information Processing Standards — are publicly announced standards developed by the National Institute of Standards and Technology (NIST). Chainguard offers images that use FIPS-validated cryptographic software modules to help users ensure that their applications meet FIPS standards.
+
+Chainguard offers FIPS versions of many of its container images, so FIPS images will fall into more than one category. Some Chainguard images with FIPS variants are [nginx](https://images.chainguard.dev/directory/image/tensorflow/overview), [PHP](https://images.chainguard.dev/directory/image/php-fips/overview), and [PyTorch](https://images.chainguard.dev/directory/image/pytorch-fips/overview). 
+
+For more information, please refer to our conceptual article on [Chainguard FIPS Images](/chainguard/chainguard-images/features/fips/fips-images/).
+
+
+## Image Type Considerations
+
+There is some overlap between the different image categories outlined in this guide. For example, the [PyTorch](https://images.chainguard.dev/directory/image/pytorch/overview) image is an AI image, but it is also part of our free tier, meaning it's also a Starter image. 
```

Comment:
Double check throughout that the capitalization of things like "Starter image" here are consistent across the page and with what we have in the style guide. I think several of these terms are new in our docs, so let's get them right from the start.